### PR TITLE
chore: improved error output of `psximager.Execute`

### LINKED ISSUE #26: Improve error output of `psximager.Execute`
Namely, only the output of the `cmd` command (e.g. `error executing psxinject: exit status 1`) is decorated, instead of the whole stderr of the actual `psxinject` command.
For this, `cmd.Stdout` and `cmd.Stderr` should be readable instead just writing to their `os` equivalent readers.
Can be reproduced using `ExampleExecute` in `psxinject_test.go`.

I first about doing all of this using a logger (preferably [the one in the experimental slog package](https://pkg.go.dev/golang.org/x/exp/slog#Logger)) or making this configurable, but then decided against that, because the whole `psximager` package is internal anyways.

### PR DESCRIPTION
**Pull request by ChristoWolf**
Resolves #26.

### CODE REVIEW

**Review by AmenAnthem — APPROVED** [review]
LGTM

**Comment by ChristoWolf** [comment]
~~Need to increase coverage.~~
[Done.](https://github.com/ChristoWolf/vagrant-story-randomizer/commit/c2565077efef918dff8f4b871b64b3b7cfbd82ea)

@AmenAnthem: Don't let the pipeline failure fool you, that's a codecov bug.
Executing the action again might make it green.

File: internal/psximager/psxinject.go [hunk 1]
```diff
@@ -25,7 +26,10 @@ func newInjectCmd(psximagerPath, cueFile, orgFile, injectFile string) *exec.Cmd
 func Execute(psximagerPath, cueFile, orgFile, injectFile string) error {
 	cmd := newInjectCmd(psximagerPath, cueFile, orgFile, injectFile)
 	if err := cmd.Run(); err != nil {
-		return fmt.Errorf("error executing %s: %w", PsxInject, err)
+		if stderr, ok := cmd.Stderr.(*bytes.Buffer); ok {
```
Comment (ChristoWolf) [inline internal/psximager/psxinject.go:29]: Needs to be covered.

Comment (ChristoWolf) [reply internal/psximager/psxinject.go:29]: [Done.](https://github.com/ChristoWolf/vagrant-story-randomizer/commit/c2565077efef918dff8f4b871b64b3b7cfbd82ea)

**Pull request merged**